src/schema/v2/homeView/sectionTypes/Tasks.ts

```diff
@@ -0,0 +1,32 @@
+import { GraphQLObjectType } from "graphql"
+import { pageable } from "relay-cursor-paging"
+import { connectionWithCursorInfo } from "schema/v2/fields/pagination"
+import { TaskType } from "schema/v2/me/task"
+import { NodeInterface } from "schema/v2/object_identification"
+import { ResolverContext } from "types/graphql"
+import {
+  HomeViewGenericSectionInterface,
+  standardSectionFields,
+} from "./GenericSectionInterface"
+import { HomeViewSectionTypeNames } from "./names"
+
+export const HomeViewTasksSectionType = new GraphQLObjectType<
+  any,
+  ResolverContext
+>({
+  name: HomeViewSectionTypeNames.HomeViewSectionTasks,
+  description: "A tasks section in the home view",
+  interfaces: [HomeViewGenericSectionInterface, NodeInterface],
+  fields: {
+    ...standardSectionFields,
+
+    tasksConnection: {
+      type: connectionWithCursorInfo({
+        nodeType: TaskType,
+      }).connectionType,
+      args: pageable({}),
+      resolve: (parent, ...rest) =>
+        parent.resolver ? parent.resolver(parent, ...rest) : [],
```

Comment:
**suggestion**: As discussed in [this PR](https://github.com/artsy/metaphysics/pull/6117), we should fallback to an [empty connection shape](https://github.com/artsy/metaphysics/blob/0f2e865fbe86b705cace2df46018daf1c7dedaec/src/schema/v2/fields/pagination.ts#L235-L238) here as an empty array doesn't match the expected type.